interpreter/cling/tools/packaging/cpt.py

```diff
@@ -239,11 +239,13 @@ def download_llvm_binary():
         if llvm_config_path != '' and tar_required is False:
             llvm_dir = os.path.join("/usr", "lib", "llvm-"+llvm_vers)
             if llvm_config_path[-1:] == "\n":
-                llvm_config_path = llvm_config_path[:-1]
+                llvm_config_path = llvm_config_path[:-1]            
             llvm_flags = "-DLLVM_BINARY_DIR={0} -DLLVM_CONFIG={1} -DLLVM_LIBRARY_DIR={2} -DLLVM_MAIN_INCLUDE_DIR={3} -DLLVM_TABLEGEN_EXE={4} \
                           -DLLVM_TOOLS_BINARY_DIR={5} -DLLVM_TOOL_CLING_BUILD=ON".format(llvm_dir, llvm_config_path,
                            os.path.join(llvm_dir, 'lib'), os.path.join(llvm_dir, 'include'), os.path.join(llvm_dir, 'bin', 'llvm-tblgen'),
                            os.path.join(llvm_dir, 'bin'))
+            if args['verbose']:
+                llvm_flags += "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
```

Comment:
```suggestion
                llvm_flags += " -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON "
```
Don't we need that to concatenate with the rest?